aep/general/0162/aep.md.j2

```diff
@@ -127,6 +125,8 @@ Services **may** reserve specific IDs to be [aliases][alias] (e.g.
 GET /v1/publishers/{publisher_id}/books/{book_id}/revisions/{revision_id}
 ```
 
+If specific IDs are reserved, services **should** document those IDs.
```

Comment:
Why not must?

Reply:
sure, isn't really an enforceable must though.

Reply:
added